opm/autodiff/NonlinearSolver_impl.hpp

```diff
@@ -103,11 +103,11 @@ namespace Opm
             }
             linIters += report.linear_iterations;
             ++iteration;
-        } while ( (!converged && (iteration <= maxIter())) || (minIter() > iteration));
+        } while ( (!converged && (iteration <= maxIter())) || (iteration <= minIter()));
```

Comment:
Do you really mean `iteration <= minIter()` here?  Doesn't that imply that the very minimum number of iterations we can ever have is `minIter() + 1`?